crates/stark-felt/src/lib.rs

```diff
@@ -124,7 +130,7 @@ impl Felt {
         BitArray::new(limbs)
     }
 
-    /// Checks if `self` is equal to [Felt::Zero].
+    /// Checks if `self` is equal to [Felt::ZERO].
     pub fn is_zero(&self) -> bool {
```

Comment:
Why are we not using the https://docs.rs/num/latest/num/traits/trait.Zero.html trait?
We are importing num-traits in this crate, so we have access to it. Is it a deliberated choice?